### PR TITLE
Ncnews 6

### PR DESCRIPTION
**Pull request by Controllah-Gabi**

### CODE REVIEW

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Hey Controllah :) 
Please...
-Make sure your comments are on a key of comments, not comment (I know this is picky).
-Your forEach will require a .length check to avoid false positives.
-Make a test for an existing article that has no comments. this will require you to make a second pool.query to check the article exists before selecting comments by ID.

Let us know if you have any issues :)

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Great :) 
Please could you add a test for an article that exists but has 0 comments. then you should be good 👍

File: __tests__/app.test.js [hunk 1]
```diff
@@ -101,4 +101,55 @@ describe("5. GET /api/articles/:article_id",()=>{
...
+
+        test("should return error message if article does not exist. 999 test",()=>{
+            return request(app)
+            .get("/api/articles/999/comments")
+            .expect(404)
+            .then(data=>{
+                expect(data.body.msg).toBe("Invalid ID")
+            })
+        })
+    })
```
Comment (emilyb93) [inline __tests__/app.test.js:164]: Id like to see a test for if the review_id is not a number but the rest of the URL is fine. Soemthing like `/api/articles/sports/comments`

**Review by emilyb93 — CHANGES_REQUESTED** [review]
One more test please

**Review by hevdevs — APPROVED** [review]
Nice! good to merge

**Pull request merged**